test/api_test.js

```diff
@@ -21,14 +21,14 @@ suite('API', () => {
     await namespaces.removeTable();
   });
 
-  test('ping', () => {
+  test.skip('ping', () => {
```

Comment:
It looks like `api` does not have a ping method anymore. For now, I just skipped this test, but if this functionality is not necessary anymore, I think the `ping` test is no longer necessary.